src/services/ipfs-service.ts

```diff
@@ -116,11 +117,16 @@ export class IpfsServiceImpl implements IpfsService {
   /**
    * Sets the record and returns its CID
    * @param record - Record value
+   * @param pin - Add to the pin store
    */
-  public async storeRecord(record: Record<string, unknown>): Promise<CID> {
+  public async storeRecord(record: Record<string, unknown>, pin = true): Promise<CID> {
```

Comment:
why even have this argument if we never set it to false?

Reply:
I made it opt-in to store the metadata only, then switched this argument to pin by default.
I changed the logic to always pin, without argument.